### DISCUSSION TITLE
How is this creating snapshots from UTM's raw images?

### DISCUSSION THREAD

**Discussion by dataprolet** [Metamogul/UTM-Snapshot-Manager#4] [General]
I'm just curious how to create a manual snapshot from a raw image in a UTM container? When I try to use the following command manually I always get an error. I'd appreciate any clarification on this. Thanks in advance!

Command I use:
`qemu-img snapshot -c $tag /User/$username/Library/Containers/com.utmapp.UTM/Data/Documents/macOS.utm/Data/$id.img`

Error I get:
`qemu-img: Could not create snapshot '$tag': -45 (Operation not supportetd)`

**Comment by Metamogul** [comment]
If `$tag` contains a value the command doesn't look so wrong. However according to documentation only the `qcow2` file format is supported. You can use the `qemu-img convert` subcommand to convert an image to the `qcow2` format. Does that solve your problem?
